data/language/en-GB.txt

```diff
@@ -3731,6 +3731,21 @@ STR_6280    :{SMALLFONT}{BLACK}Chat
 STR_6281    :{SMALLFONT}{BLACK}Show a separate button for the Chat window in the toolbar
 STR_6282    :Chat
 STR_6283    :Chat not available at this time. Are you connected to a server?
+STR_6284    :Network
+STR_6285    :Network Information
+STR_6286    :Receive
+STR_6287    :Send
+STR_6288    :Total received
+STR_6289    :Total sent
+STR_6290    :Base protocol
+STR_6291    :Commands
+STR_6292    :Map
+STR_6293    :B
+STR_6294    :KB
```

Comment:
Elsewhere in the codebase, we use `KiB`, `MiB`, …
(which means "binary" kilobyte, or 1024 bytes, unlike the ambiguous kB, MB, …)